lib/fathead/npm_cli/parse.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import requests
+import re
+from bs4 import BeautifulSoup
+
+
+START_URL = 'https://docs.npmjs.com/cli/install'
+BASE_URL = 'https://docs.npmjs.com'
+OUTPUL_FILE = open('output.txt', 'w+')
+RESULT_URLS = []
+
+
+# Print the content to the output file
+def print_article_line(title, code, content):
+
+    title = 'npm-' + title
+    code = code.replace('\n', '\\n').replace('\t', '  ')
+    code = code.replace('<', '&lt;').replace('>', '&gt;')
+    content = content.replace('\n', '\\n').replace('\t', '  ')
+    content = content.replace('<', '&lt;').replace('>', '&gt;')
+
+    abstract = '<section class="prog__container"><pre><code>' + code + \
+               '</code></pre>' + content + '</section>'
```

Comment:
My intuition is telling me to put the content before the code as that is probably the information the developer is searching for. Can we swap these around?